python/py-tensorflow/Portfile

```diff
@@ -49,23 +49,19 @@ xcode_workaround.os_versions           18 19
 xcode_workaround.fixed_xcode_version   11.4
 
 # Supported python versions
-python.versions  37 38 39
+python.versions  37 38 39 310
 
 if {${name} ne ${subport}} {
-
-    PortGroup bazel 1.0
-
-    # See https://www.tensorflow.org/install/source#macos for tested bazel versions.
-    bazel.version   3.7
+    PortGroup       bazel 1.0
 
     # TF versions 1 and 2 cannot be installed at once
-    conflicts py${python.version}-tensorflow1
+    conflicts       py${python.version}-tensorflow1
 
-    # Conflicts with tensorflow-macos
-    if { (${build_arch} eq {arm64} && (${python.version} eq 38 || ${python.version} eq 39))
-        || (${build_arch} eq {x86_64} && ${python.version} eq 38) } {
+    # Conflicts with tensorflow-macos, tensorflow-metal
+    if { ${python.version} eq 38 || ${python.version} eq 39 || ${python.version} eq 310 } {
         conflicts-append \
                     py${python.version}-tensorflow-macos \
+                    py${python.version}-tensorflow-metal \
                     py${python.version}-tensorflow_macos
```

Comment:
Given that `py-tensorflow_macos` is obsolete, let's remove that conflict declaration.

Reply:
> Given that `py-tensorflow_macos` is obsolete, let's remove that conflict declaration.

Done. Also deleted that port, along with removing all references to it.

Reply:
> > Given that `py-tensorflow_macos` is obsolete, let's remove that conflict declaration.
> 
> Done. Also deleted that port, along with removing all references to it.

No need to wait for this PR, so I took of deletion on master. Also rebased to pick up those changes.